### ISSUE TITLE
Non-Windows versions?

### ISSUE THREAD

**Issue by dilworks** [Jessecar96/SteamDesktopAuthenticator#57]
First, thanks for your efforts (and piss off Valve, I don't want a smartphone in my life EVER)

But then, I'm not running Windows, but Linux. Yes, I know that we have Mono which can run a lot of .NET applications, but since yours is made in .NET 4.5, dunno how it would roll in Mono.

Is there any hope for non-Windows targets? Or at least, has anyone tested it on Mono?

**Comment by Jessecar96** [comment]
It's been tested to work on wine, not sure about mono yet.

**Comment by Aidoboy** [comment]
Yeah, it works great, at least in wine-staging. http://www.wine-staging.com/

**Comment by epsimatic** [comment]
Launches, but hangs on login on wine-1.8 on OS X via WineSkin

**Comment by Aidoboy** [comment]
I was using Wine 1.9 vie PlayonLinux on Ubuntu when it worked for me.

**Comment by dyc3** [comment]
I made a CLI program to replace SteamDesktopAuthenticator on Linux, [steamguard-cli](https://github.com/dyc3/steamguard-cli). It can read and write the same files that SteamDesktopAuthenticator can, so you don't have to set up your mobile authenticator again.
